hplsql/src/main/java/org/apache/hive/hplsql/udf/Udf.java

```diff
@@ -66,48 +64,25 @@ public ObjectInspector initialize(ObjectInspector[] arguments) throws UDFArgumen
     if (!(arguments[0] instanceof StringObjectInspector)) {
```

Comment:
Now it requires minimum two arguments so validation can be changed. Also second argument should be StringObjectInspector type so this check can also be added.

Reply:
Thanks for pointing this out! I will update this and also the check just before this to check for minimum 2 arguments.